src/pool-cl/CLPositionManager.sol

```diff
@@ -99,42 +81,6 @@ contract CLPositionManager is
         _;
     }
 
-    /// @notice Reverts if the hash of the config does not equal the saved hash
-    /// @param tokenId the unique identifier of the ERC721 token
-    /// @param config the PositionConfig to check against
-    modifier onlyValidConfig(uint256 tokenId, PositionConfig calldata config) override {
-        if (positionConfigs[tokenId].getConfigId() != config.toId()) revert IncorrectPositionConfigForTokenId(tokenId);
-        _;
-    }
-
-    /// @inheritdoc ICLPositionManager
-    function positions(uint256 tokenId)
-        external
-        view
-        returns (
-            PoolKey memory poolKey,
-            int24 tickLower,
-            int24 tickUpper,
-            uint128 liquidity,
-            uint256 feeGrowthInside0LastX128,
-            uint256 feeGrowthInside1LastX128
```

Comment:
is there a way for us to lookup the `liquidity  | feeGrowthInside0LastX128 | feeGrowthInside1LastX128` if this is merged?

Reply:
ah saw it below, deleting this comment